### ISSUE TITLE
Poor Result on Custom Training Dataset

### ISSUE THREAD

**Issue by williambittner1** [NVlabs/neuralangelo#103]
I have trained on a custom dataset (349 images from an indoor office scene), however the results (depth map, normal map, extracted mesh geometry) are very poor. (RGB seems quite good though). The loss and psnr graphs also seem quite suspicious. I followed your data-preprocessing instructions and the colmap-poses and points seem to be good. I also also set SCENE_TYPE=indoor during data preprocessing. Can you help me find the problem?  

<img width="959" alt="Screenshot 2023-09-02 150926" src="https://github.com/NVlabs/neuralangelo/assets/72436975/564d0d3f-ae3c-42bf-99db-f78da866b78d">
<img width="958" alt="Screenshot 2023-09-02 150947" src="https://github.com/NVlabs/neuralangelo/assets/72436975/ea41c8ef-0985-4b42-a8e8-d1b9fdb4c40b">
<img width="940" alt="Screenshot 2023-09-02 150852" src="https://github.com/NVlabs/neuralangelo/assets/72436975/7d9cfe35-b213-4c7e-8dfe-1057894e254d">
<img width="964" alt="Screenshot 2023-09-02 151020" src="https://github.com/NVlabs/neuralangelo/assets/72436975/b9655426-8ff1-439a-a327-87eb81b4acdd">
<img width="980" alt="Screenshot 2023-09-02 150812" src="https://github.com/NVlabs/neuralangelo/assets/72436975/c4443889-1221-49d6-a4b2-bab33a0fb118">
<img width="1304" alt="Screenshot 2023-08-31 132048" src="https://github.com/NVlabs/neuralangelo/assets/72436975/5e3464fb-fe55-441a-853f-ab5484c27fa1">

**Comment by rafaepires** [comment]
I have a same problem...
williambittner1, do you find a solution?
best wishes.

**Comment by williambittner1** [comment]
I haven't fully solved it yet. However, I now get some better results after rerunning colmap manually and changing scene_type to object. Next, I will probably try increasing the scene_scale to 2 (cfg.data.readjust.scale = 2. inside generate_config.py) instead of 1 and run again with scene_type "indoor". Let's see...

**Comment by raficabral** [comment]
@williambittner1 did you have better output?
